web/src/components/planner/PlannerFactoryExports.vue

```diff
@@ -62,8 +62,8 @@
                   color="green"
                   size="small"
                   variant="flat"
-                  @click="fixShortage(factory, getProduct(factory, exportItem.productId))"
-                >Fix production</v-btn>
+                  @click="fixExport(factory, exportItem.productId)"
+                >Fix Production</v-btn>
```

Comment:
Changed this as we're actually fixing the Production, rather than the export. Otherwise the user might get a bit confused.